notification-portlet-webapp/src/main/java/org/jasig/portlet/notice/service/jpa/INotificationDao.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Licensed to Jasig under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work
+ * for additional information regarding copyright ownership.
+ * Jasig licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a
+ * copy of the License at:
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.jasig.portlet.notice.service.jpa;
+
+import java.util.List;
+import java.util.Set;
+
+import org.jasig.portlet.notice.NotificationState;
+
+/* package-private */ interface INotificationDao {
```

Comment:
I didn't immediately understand why the `.jpa` package would need a package-private `DAO` interface that returns `JpaEntry` objects.  JavaDoc might help articulate what this interface is intended for.